docs/billing-integration.md

```diff
@@ -138,6 +167,11 @@ data:
     # (defined by 'intervalSeconds'); this is assuming that the metric
     # configured in service control has 'seconds' as its unit.
     #
+    # This section is only a convenience for reporting instance
```

Comment:
Does this section just repeat the paragraph above? That heartbeat is the only supported metric?

Reply:
It does - removed an instead added to the first sentence in the prior paragraph to emphasize when this section should be omitted:

```
    # The sources section lists metric data sources run by the agent
    # itself - omit it if usage reports for all metrics are reported by
    # the application.
```